manifests/pod.yaml

```diff
@@ -12,7 +12,7 @@ spec:
   serviceAccountName: jenkins
   containers:
    - name: jnlp
-     image: quay.io/openshift/origin-jenkins-agent-base:latest
+     image: JENKINS_AGENT_IMAGE
```

Comment:
I forgot to mention this, but this actually also fixes a mismatch issue. Before we were using an unversioned agent image from Quay. Now, we're following the in-cluster imagestream, just like for the controller itself, so we know they're always matching.